src/widgets/user-settings.tsx

```
@@ -0,0 +1,412 @@
+  const [open, setOpen] = useState(false);
+
+  const localLanguage = localUserSettings.videoLanguage;
+
+  return (
+    <div className="w-full">
+      <Popover open={open} onOpenChange={setOpen}>
+        <PopoverTrigger asChild>
+          <Button
+            variant="outline"
+            // biome-ignore lint/a11y/useSemanticElements: ShadCN semantics
+            role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)